docs/user/azure/install_upi.md

```diff
@@ -194,16 +194,10 @@ export ACCOUNT_KEY=`az storage account keys list -g $RESOURCE_GROUP --account-na
 Given the size of the RHCOS VHD, it's not possible to run the deployments with this file stored locally on your machine.
 We must copy and store it in a storage container instead. To do so, first create a blob storage container and then copy the VHD.
 
-Choose the RHCOS version you'd like to use and export the URL of its VHD to an environment variable. For example, to use the latest release available for the 4.3 version, use:
+For example, to use the version of the openshift-installer bootimages, use:
```

Comment:
I recommend we omit this entirely. We are not providing an *example* way of obtaining the VHD URL. We are providing the one recommended way of obtaining the VHD URL.

Let's combine the export of the VHD_URL environment variable in with the shell code for copying to the blob.